catalogue/webapp/pages/search/index.tsx

```diff
@@ -198,28 +192,26 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
     const defaultProps = removeUndefinedProps({
       serverData,
       query,
-
-      // TODO Harrison to explore what properties we'd want here
```

Comment:
Have you chatted with Harrison, or did this TODO get deleted erroneously?

Reply:
Yes just this morning - he's meant to close his ticket and we'll go ahead with these properties.